src/plat/tqma8xqp1gb/config.cmake

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2021, Breakaway Consulting Pty. Ltd.
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(tqma8xqp1gb KernelPlatformTqma8xqp1gb PLAT_TQMA8XQP1GB KernelArchARM)
+
+if(KernelPlatformTqma8xqp1gb)
+    declare_seL4_arch(aarch64)
+    set(KernelArmCortexA35 ON)
+    set(KernelArchArmV8a ON)
+    config_set(KernelARMPlatform ARM_PLAT ${KernelPlatform})
+    set(KernelArmVtimerUpdateVOffset OFF)
+    set(KernelArmDisableWFIWFETraps ON)
+    list(APPEND KernelDTSList "tools/dts/${KernelPlatform}.dts")
```

Comment:
Could merge this
```suggestion
    list(APPEND KernelDTSList "tools/dts/${KernelPlatform}.dts" "src/plat/tqma8xqp1gb/overlay-${KernelPlatform}.dts")
```

Reply:
The multiple lines seems to be the pattern in every other `config.cmake` file.

I believe the reason is avoiding line length, but I don't know for sure.

I have no strong feelings one way or the other, but I'd suggest we should be consistent.